sklearn/tests/test_common.py

```diff
@@ -63,8 +64,12 @@ def test_non_meta_estimators():
             continue
         if name.startswith("_"):
             continue
-        for check in _yield_all_checks(name, Estimator):
-            yield _named_check(check, name), name, Estimator
+        estimator = Estimator()
+        # check this on class
+        yield check_no_fit_attributes_set_in_init, name, Estimator
```

Comment:
Right now the tests for meta-estimators are not very strong. They will be much stronger with the estimator tags branch merged. I tried to keep this PR small, it doesn't increase test coverage much, but it doesn't decrease it anywhere afaik.